runtime/dart_isolate.cc

```diff
@@ -211,10 +211,12 @@ std::weak_ptr<DartIsolate> DartIsolate::CreateRootIsolate(
               )));
 
   auto isolate_data = std::make_unique<std::shared_ptr<DartIsolate>>(
-      std::shared_ptr<DartIsolate>(new DartIsolate(settings,  // settings
-                                                   true,      // is_root_isolate
-                                                   context    // context
-                                                   )));
+      std::shared_ptr<DartIsolate>(new DartIsolate(
+          settings,           // settings
+          true,               // is_root_isolate
+          context,            // context
+          !!spawning_isolate  // spawn_in_group
```

Comment:
C++ style guide has parameter comments in the form of `/*spawn_in_group=*/!!spawning_isolate`.  Comments like that are checked by the linter.

Reply:
Done